react/components/Button/index.tsx

```diff
@@ -41,7 +41,7 @@ const propTypes = {
   /** (Button spec attribute) */
   autoComplete: PropTypes.string,
   /** (Button spec attribute) */
-  disabled: PropTypes.bool,
+  disabled: PropTypes.bool.isRequired,
```

Comment:
when they use the button, is it necessary to change it to prop disabled?

Reply:
Nope, but disabled and icon is by default props false. If this props isn't required, eslint will warn to fix this [line](https://github.com/vtex/styleguide/blob/375d01193da371167c2e5a861024bdab43eb2f6f/react/components/Button/index.tsx#L162). The eslint fix changes from `||` to `??` operators, but these two operators aren't equivalents.

Reply:
but it is not obliging the user to pass the `disabled` and `icon` props to the button that will resolve this.

Reply:
Ok, so can I remove it from default props?